framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/DatabaseDeployment.java

```diff
@@ -27,7 +28,7 @@
      *
      * @return Database URL.
      */
-    URL getUrl();
+    Optional<URL> getUrl();
```

Comment:
What is the reason for this change? Is database URL used anywhere?

Reply:
To return URL in same type as for other deployments. If URL for DB is not needed, we can remove this method.